sympy/functions/elementary/complexes.py

```diff
@@ -395,8 +395,8 @@ class Abs(Function):
     >>> abs(-x) # The Python built-in
     Abs(x)
 
-    Note that the Python built-in will return either an Expr or int depending on
-    the argument::
+    Note that the Python built-in will return either an Expr or int depending
+    on the argument::
```

Comment:
Personally, it **really** helps me when the lines are <= 79 char long. There's a reason that my editor works like that - and there's a reason why PEP8 for that exists.